starknet_py/tests/e2e/deploy/deployer_test.py

```diff
@@ -98,26 +98,26 @@ async def test_constructor_arguments_contract_deploy(
 @pytest.mark.parametrize(
     "salt, pass_account_address", [(1, True), (2, False), (None, True), (None, False)]
 )
-async def test_address_computation(
-    salt, pass_account_address, gateway_account, map_class_hash
-):
+async def test_address_computation(salt, pass_account_address, account, map_class_hash):
     deployer = Deployer(
-        account_address=gateway_account.address if pass_account_address else None
+        account_address=account.address if pass_account_address else None
     )
+    if isinstance(salt, int) and isinstance(account.client, FullNodeClient):
+        # transactions have to be different for each account
+        salt = salt + 1
```

Comment:
```suggestion
        salt += 2
```
One of the `salt` is `1`, so after adding 1 it will be `2` as a second case